include/gsl/multi_span

```diff
@@ -1494,7 +1494,9 @@ public:
 
     constexpr reference operator[](const index_type& idx) const GSL_NOEXCEPT
     {
-        return data_[bounds_.linearize(idx)];
```

Comment:
@neilmacintosh; No idea. Apparently I wasn't fully awake when makeing this change ;) Sorry for that. I removed the last two commits as they where useless without compiler support for attribute syntax anyway.